docs/home/contribute/create-pull-request.md

```diff
@@ -44,12 +44,30 @@ choose the
 [page type](/docs/home/contribute/page-templates/)
 that is the best fit for your content.
 
+## Spell Checking
+
+Check the spelling of any proposed changes. Verify any changes made by an automated spellchecker.
+
+You can check typos with the help of [Misspell](https://github.com/client9/misspell), which has been used in Kubernetes and Community project.
+
+1. If you use Go, you can install the Misspell tool by entering this command:
+    ```
+    go get -u github.com/client9/misspell/cmd/misspell
+    ```
+   Or you can install the tool [in other ways](https://github.com/client9/misspell#install).
+
```

Comment:
Go getting misspell is not enough to run it. You have to build it first. So I think we need to show that as a step.

1. Get misspell.
2. Build misspell.
3. Run misspell.

And we need to mention that misspell has to be on your path. Either that or you have to specify the full path to the misspell command: ... | xargs ~/bin/missspell

Reply:
`go get` command should do install itself completely. But maybe we can't list all the solutions in this document(like all the chinese contributor should set proxy...). So i add a short description and a link.

Reply:
Thanks, I didn't realize that the go get command did an installation.